File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Conversions.java
package org.firstinspires.ftc.teamcode.util;

public class Conversions {
    public static double mmToInches(double mm) {
        return mm / mmPerInch;
    }
    public static double inchesToMm(double inches) {
        return inches * mmPerInch;
    }
    public static final float mmPerInch        = 25.4f;
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Conversions.kt
package org.firstinspires.ftc.teamcode.util

object Conversions {
    fun mmToInches(mm: Double): Double {
        return mm / mmPerInch
    }

    fun inchesToMm(inches: Double): Double {
        return inches * mmPerInch
    }

    val mmPerInch = 25.4f
}